docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,76 @@
+# Flask-k8s Charm Tutorial
+
+## What You'll Do
+
+In this tutorial, we'll work on:
+
+- Constructing a Flask container
+- Deploying the [flask-k8s charm](https://charmhub.io/flask-k8s)
+
+The flask-k8s charm simplifies the deployment of scalable Flask applications,
+in addition to facilitating charm operation by interfacing with the Canonical
+Observability Stack (COS). We'll guide you step-by-step through the entire
+deployment process to set up a basic Flask deployment.
+
+## Prerequisites
+
+For deploying the flask-k8s charm, ensure you have a juju installation
+bootstrapped with any kubernetes controller. To bootstrap your juju
+installation with microk8s, refer to the microk8s [installation
+documentation](https://juju.is/docs/olm/microk8s).
+
+## Setting up a Tutorial Model
+
+To manage resources effectively and to separate this tutorial's workload from
+your usual work, we recommend creating a new model using the following command.
+
+```
+juju add-model flask-tutorial
+```
+
+## Building the Flask Container
+
+Unlike some other Kubernetes charms, the flask-k8s charm operates in a 'bring
+your own container' mode, meaning it doesn't provide a workload container. A
+container image for use with the flask-k8s charm must meet the following
+requirements:
+
+- It should have gunicorn installed as a global Python package
+- The Flask application should be situated in the `/srv/flask/app` directory
+- The Flask WSGI application should be located at `/srv/flask/app/app.py`, with
+  the variable name `app`
+- The Flask application must invoke `app.config.from_prefixed_env()` to receive
+  charm configuration
```

Comment:
Perhaps remove charm and just say "to receive configuration"?

Reply:
Fixed, thanks!